scripts/bc-wp-init.sh

```diff
@@ -4,13 +4,31 @@ cp ./provision/default.yml ./site.yml
 printf "Domain name (eg. promocode.co.ke): "; read domain
 sed -i -e "/hostname:/s/.*/hostname: $domain/" site.yml
 
-echo -n "Do you want to use PHP 5.6 instead of PHP 7.0 (y/N)?"
-read php_ans
-if echo "$php_ans" | grep -iq "^y"; then
-    sed -i -e "/enable_php56:/s/.*/enable_php56: true/" site.yml
-else
-    sed -i -e "/enable_php56:/s/.*/enable_php56: false/" site.yml
-fi
+printf "Specify version of PHP (press number)\n1) PHP 5.6;\n2) PHP 7.0  <-- default value \n3) PHP 7.1\n4) PHP 7.2\n5) PHP 7.3\nEnter number (PHP 7.0 as default): "; read php_version_number
```

Comment:
`press number` not sufficient. "Use list number, eg. `1`" . Or something similar so it is clear for engineers.